SLAM/iterative_closest_point/iterative_closest_point.py

```diff
@@ -35,15 +36,24 @@ def icp_matching(previous_points, current_points):
         count += 1
 
         if show_animation:  # pragma: no cover
```

Comment:
Could you please extract these plot related code inside the if clause to a function like `plot_points`?